scripts/component-generator/src/transforms/stories-file.js

```diff
@@ -7,7 +7,10 @@ module.exports = (file, api, options) => {
 
   j(imports[imports.length - 1]).insertAfter(
     `// This import was added by the component generator
-import './${ComponentName}/index.story';`,
+import './components/${ComponentName}/index.story';
```

Comment:
shouldn't these point to `src`?
```suggestion
import '../src/${ComponentName}/docs/index.story';
```

Reply:
fixed